web_src/js/components/RepoContributors.vue

```diff
@@ -384,7 +390,7 @@ export default {
               {{ contributor.name }}
             </h4>
             <p class="gt-font-12 gt-df gt-gap-2">
-              <strong v-if="contributor.total_commits">{{ contributor.total_commits.toLocaleString() }} {{ locale.contributionType.commits }}</strong>
+              <strong v-if="contributor.total_commits"><a :href="searchQuery + contributor.email">{{ contributor.total_commits.toLocaleString() }} {{ locale.contributionType.commits }}</a></strong>
```

Comment:
What about users having multiple emails?
For those, only the primary email will be counted if I see that correctly…